copilot/code.py

```
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, Depends, HTTPException
+from kubernetes import client, config
+
+app = FastAPI()
+
+def get_k8s_client():
+    """Load Kubernetes configuration and return the API client."""
+    try:
+        config.load_kube_config()  # Loads from ~/.kube/config or in-cluster config
+    except client.config_exception.ConfigException as e:
+        raise HTTPException(status_code=500, detail=f"Failed to load Kubernetes config: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)